### PR TITLE
Enable televore again

### PR DESCRIPTION
**Pull request by ReddeyfishVor**

### CODE REVIEW

**Comment by AnonymousHybi** [comment]
WHY

File: code/datums/helper_datums/teleport.dm [hunk 1]
```diff
@@ -188,7 +188,7 @@
 	//VOREStation Edit Start
 	if(destination.z in using_map.player_levels)
 		return 1
-	if(istype(teleatom, /mob/living))
-		to_chat(teleatom, "<span class='warning'>The portal refuses to carry you that far away!</span>")
+	//if(istype(teleatom, /mob/living))
```
Comment (AnonymousHybi) [inline code/datums/helper_datums/teleport.dm]: Do a check to see if their belly has over 3 mobs.

If there's over 3 mobs then refuse to teleport.

There's so many things I dont recommend with televore it's so fustrating to balance. It's why they disabled it originally.

Comment (ReddeyfishVor) [reply code/datums/helper_datums/teleport.dm]: Will do. Do you mean mobs, or just items?

**Comment by Arokha** [comment]
Was originally disabled because it was too difficult to set up events that you transit to via teleports. Like if you have something like the tcommsat or an admin-loaded map for an event, all the people teleporting at once basically assures that they'll eat each other and then have to spend the next few minutes un-pref-breaking or whatever.

**Comment by ReddeyfishVor** [comment]
Hmm. Well, we haven't had any of those events and we don't have any planned for the near future, so I'll enable it on our server for now. Thank you for the insight!

**Pull request merged**